libkineto/src/CuptiActivityProfiler.h

```diff
@@ -170,7 +170,7 @@ class CuptiActivityProfiler {
     if (resourceInfo_.find({pid, tid}) == resourceInfo_.end()) {
       resourceInfo_.emplace(
           std::make_pair(pid, tid),
-          ActivityLogger::ResourceInfo(
+          libkineto::ResourceInfo(
```

Comment:
isn't it already in libkineto namespace, may not be right on this . cc @aaronenyeshi there was some detail about KINETO_NAMESPACE that I'm missing here.

Reply:
thx, removed it